plugins/outputs/groundwork/groundwork.go

```diff
@@ -254,12 +260,18 @@ func (g *Groundwork) parseMetric(metric telegraf.Metric) (string, *transit.Dynam
 		if err != nil {
 			return "", nil, err
 		}
+		if typedValue.ValueType == transit.StringType {
+			g.Log.Warn("string values are not supported, skipping")
+			continue
+		}
```

Comment:
Not sure if this is the preferred way to go for telegraf, will discuss that internally.

Reply:
I would definitely mention this in the README.

Reply:
I asked Thomas Stocking about this, but he said that all we can do at the moment is avoid metrics that we cannot handle on the server side.

Reply:
I was just talking about if telegraf maintainers would accept if an output just 'silently' ignores a field instead of erroring-out. Will ask them about that.

Reply:
In case of erroring-out we are losing another metrics from the batch. We can add information to the logs(with warn level) to notify users about skipping string metrics.

Reply:
Yeah, I know. That's why I can't make a decision on that by myself.

Reply:
Ok, let me know about the results of the conversation.
Thanks!

Reply:
They agree with this 😉